gwdetchar/omega/html.py

```diff
@@ -592,7 +590,7 @@ def write_summary(
     """
     utc = tconvert(gpstime)
     page = markup.page()
-    page.h2(header)
+    page.h2(header, style='font-weight:normal;')
```

Comment:
it makes sense to me to keep the style declarations to the scss file(s), where I think they will be easier to manage in the long term. can you have a look at doing that?

Reply:
These style choices will eventually be replaced by `bootstrap-ligo` when we get around to doing that, to be consistent with the summary pages. But, to your point, that will be easier if we stick them in the scss files now, so I'll address that shortly.